crates/hc_sandbox/src/config.rs

```diff
@@ -19,7 +19,7 @@ pub fn create_config(
         dpki: DpkiConfig::disabled(),
         #[cfg(feature = "unstable-dpki")]
         dpki: DpkiConfig::testing(),
-        ..Default::default()
+        ..ConductorConfig::empty()
```

Comment:
`default` returns `empty` anyway and is more general.
```suggestion
        ..Default::default()
```

Reply:
Removed the empty config, reverted back to using default